### PR TITLE
Workday

### PR DESCRIPTION
**Pull request by Bargenish**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: demisto/etc/issues/9539

## Description
Integration with Workday

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [x] 5.0.0
- [X] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [ ] Tests
- [x] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

File: Packs/Workday/Integrations/Workday/Workday.py [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+    except Exception as e:
+        raise DemistoException(
+            f"Test failed. Try checking the credentials you entered. \n {e}")
+    return 'ok'
+
+
+def list_workers_command(client: Client, args: dict) -> CommandResults:
+    count = int(args.get('count', "1"))
+    page = int(args.get('page', "1"))
+    num_of_managers = int(args.get('managers', "3"))
```
Comment (yaakovi) [inline Packs/Workday/Integrations/Workday/Workday.py:269]: be consistant
```suggestion
    count = int(args.get('count', '1'))
    page = int(args.get('page', '1'))
    num_of_managers = int(args.get('managers', '3'))
```

File: Packs/Workday/Integrations/Workday/Workday.py [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+            </bsvc:Response_Group>
+        </bsvc:Get_Workers_Request>
+    </soapenv:Body>
+</soapenv:Envelope>
+"""
+
+
+def convert_to_json(response):
+    raw_json_response = json.loads(xml2json(response))
+    workers_data = raw_json_response.get('Envelope').get('Body').get('Get_Workers_Response').get('Response_Data'). \
```
Comment (yaakovi) [inline Packs/Workday/Integrations/Workday/Workday.py:85]:
```suggestion
    workers_data = raw_json_response.get('Envelope', {}).get('Body', {}).get('Get_Workers_Response', {}).get('Response_Data', {}). \
```

File: Packs/Workday/Integrations/Workday/Workday.py [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+# Disable insecure warnings
+
+requests.packages.urllib3.disable_warnings()
+
+# TODO: remove before pr:
+
+
+# CONSTANTS
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+API_VERSION = "v34.0"
```
Comment (yaakovi) [inline Packs/Workday/Integrations/Workday/Workday.py:15]: in the readme you stated other version

File: Packs/Workday/Integrations/Workday/Workday.py [hunk 4]
```diff
@@ -0,0 +1,319 @@
...
+        position_data_path = worker.get('Employment_Data', {}).get('Worker_Job_Data', {}).get('Position_Data')
+        worker_status_data = worker.get('Employment_Data', {}).get('Worker_Status_Data')
+        business_site_address_data = position_data_path.get('Business_Site_Summary_Data', {}).get('Address_Data')
+        if isinstance(business_site_address_data, list):
+            business_site_address_data = business_site_address_data[-1]
+
+        worker_context = {
+            "Worker_ID": worker['Worker_ID'],
+            "User_ID": worker['User_ID'],
+            "Country": name_detail_data_path['Country_Reference']['ID'][1]['#text'],
```
Comment (yaakovi) [inline Packs/Workday/Integrations/Workday/Workday.py:109]: avoid direct access to dict fields. use `.get`

File: Packs/Workday/Integrations/Workday/Workday.yml [hunk 5]
```diff
@@ -0,0 +1,276 @@
...
+    - contextPath: Workday.Worker.Manager.Manager_Name
+      description: The manager name.
+      type: String
+  isfetch: false
+  runonce: false
+  script: '-'
+  type: python
+  subtype: python3
+  dockerimage: demisto/python3:3.8.3.8715
+fromversion: 5.5.0
```
[the remark this answers is not in the document]
Comment (Bargenish) [reply Packs/Workday/Integrations/Workday/Workday.yml:276]: changed to 5.0.0

File: Packs/Workday/Integrations/Workday/Workday.yml [hunk 6]
```diff
@@ -0,0 +1,276 @@
+category: Authentication
+commonfields:
+  id: Workday
+  version: -1
+configuration:
+- defaultvalue: https://wd5-impl-services1.workday.com/ccx/service
```
[the remark this answers is not in the document]
Comment (Bargenish) [reply Packs/Workday/Integrations/Workday/Workday.yml:6]: Global one I think - (https://learn.azuqua.com/connector-reference/workday/)

File: Packs/Workday/Integrations/Workday/Workday_description.md [hunk 7]
```diff
@@ -0,0 +1 @@
+Workday offers enterprise-level software solutions for financial management, human resources, and planning.
```
[the remark this answers is not in the document]
Comment (Bargenish) [reply Packs/Workday/Integrations/Workday/Workday_description.md:1]: waiting for @Arsenikr with this one

**Comment by kirbles19** [comment]
@Bargenish Done.

**Pull request merged**